packages/block-library/src/description-details/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+
+export const name = 'core/description-details';
+
+export const settings = {
+	title: __( 'Description Details' ),
+	parent: [ 'core/description-details' ],
+	description: __( 'The details of a term.' ),
+	category: 'layout',
+	supports: {
+		className: false,
+		lightBlockWrapper: true,
+	},
+	edit,
+	save,
+	attributes: {
```

Comment:
This, the name, and the category should be moved to a `block.json`. (Same with other blocks being added in this PR.)